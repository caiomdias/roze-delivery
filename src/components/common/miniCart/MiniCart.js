import React from 'react';
import CartIcon from '../../../assets/images/svg/cart';

const MiniCart = (props) => (
	<div className="mini-cart">
		<div className="mini-cart__counter">0</div>
		<CartIcon className="mini-cart__icon" />
	</div>
);

export default MiniCart;
