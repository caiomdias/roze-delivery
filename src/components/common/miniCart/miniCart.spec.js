import React from 'react';
import renderer from 'react-test-renderer';
import MiniCart from './MiniCart';

describe("Mini Cart Component", () => {
	it('render correctly MiniCart component', () => {  
		const miniCartComponent = renderer.create(<MiniCart />);
		expect(miniCartComponent).toMatchSnapshot();
	});
});