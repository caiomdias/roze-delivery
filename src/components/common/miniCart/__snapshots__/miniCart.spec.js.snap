// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mini Cart Component render correctly MiniCart component 1`] = `
<div
  className="mini-cart"
>
  <div
    className="mini-cart__counter"
  >
    0
  </div>
  <svg
    className="mini-cart__icon"
    onClick={null}
    role="presentation"
    viewBox="0 0 512 512"
    width="30px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M488,240H471.544a72.139,72.139,0,0,0-32.614-52.538l-6.5-123.484A7.993,7.993,0,0,0,440,56V24a8,8,0,0,0-8-8H368a8,8,0,0,0-8,8V56a7.993,7.993,0,0,0,7.569,7.978l-6.5,123.484A72.139,72.139,0,0,0,328.456,240h-.912a72.139,72.139,0,0,0-32.614-52.538l-6.5-123.484A7.993,7.993,0,0,0,296,56V24a8,8,0,0,0-8-8H224a8,8,0,0,0-8,8V56a7.993,7.993,0,0,0,7.569,7.978l-6.5,123.484A72.139,72.139,0,0,0,184.456,240h-.912a72.139,72.139,0,0,0-32.614-52.538l-6.5-123.484A7.993,7.993,0,0,0,152,56V24a8,8,0,0,0-8-8H80a8,8,0,0,0-8,8V56a7.993,7.993,0,0,0,7.569,7.978l-6.5,123.484A72.139,72.139,0,0,0,40.456,240H24a8,8,0,0,0-8,8V488a8,8,0,0,0,8,8H488a8,8,0,0,0,8-8V248A8,8,0,0,0,488,240ZM376,32h48V48H376Zm-3.261,167.108a8,8,0,0,0,4.086-6.562L383.59,64h32.82l6.765,128.546a8,8,0,0,0,4.086,6.562A56.136,56.136,0,0,1,455.417,240H344.583A56.136,56.136,0,0,1,372.739,199.108ZM232,32h48V48H232Zm-3.261,167.108a8,8,0,0,0,4.085-6.562L239.59,64h32.82l6.765,128.546a8,8,0,0,0,4.086,6.562A56.136,56.136,0,0,1,311.417,240H200.583A56.136,56.136,0,0,1,228.739,199.108ZM88,32h48V48H88ZM84.739,199.108a8,8,0,0,0,4.085-6.562L95.59,64h32.82l6.766,128.546a8,8,0,0,0,4.085,6.562A56.136,56.136,0,0,1,167.417,240H56.583A56.136,56.136,0,0,1,84.739,199.108ZM480,256V368H32V256Zm0,128v40H32V384ZM32,480V440H480v40Z"
    />
    <path
      d="M216,336h80a24,24,0,0,0,0-48H216a24,24,0,0,0,0,48Zm0-32h80a8,8,0,0,1,0,16H216a8,8,0,0,1,0-16Z"
    />
  </svg>
</div>
`;
